File: Assets/VariablesAndFunctions.js
#pragma strict

var myInt : int = 10;

function Start () {
	myInt = MultiplyByTwo(myInt);
	Debug.Log(myInt);
}

function MultiplyByTwo(number : int) : int {
	var returnValue : int;
	returnValue = number * 2;
	return returnValue;
}

File: Assets/c#/VariablesAndFunctions.cs
﻿using UnityEngine;
using System.Collections;

public class VariablesAndFunctions : MonoBehaviour {

	int myInt = 5;

	// Use this for initialization
	void Start () {
		myInt = MultiplyByTwo(myInt);
		Debug.Log(myInt);
	}

	int MultiplyByTwo(int number) {
		int ret;
		ret = number * 2;
		return ret;
	}

}
